tests/ImageSharp.Tests/Metadata/Profiles/Exif/Values/ExifValuesTests.cs

```diff
@@ -593,5 +597,19 @@ public void ExifUndefinedArrayTests(ExifTag tag)
             var typed = (ExifByteArray)value;
             Assert.Equal(expected, typed.Value);
         }
+
+        [Theory]
+        [MemberData(nameof(EncodedStringTags))]
+        public void ExifEncodedStringTests(ExifTag tag)
+        {
+            var expected = new EncodedString(EncodedString.CharacterCode.JIS, "test string");
```

Comment:
You could enumerate through the different `CharacterCode` values here and assert each one.